src/partials/header-content.hbs

```diff
@@ -53,6 +54,47 @@
         </div>
       </div>
     </div>
+  </nav>
+  {{else}}
```

Comment:
? Why do you need to duplicate the whole content of the file??? 

Did you try to just add a condition inside the nav tag to add the css class if required ? 
There are many example of how to do that, like in `page-version.hbs`
```
<a class="version
      {{~#if (eq ./version @root.page.version)}} is-current{{/if~}}
      {{~#if ./missing}} is-missing{{/if}}" href="{{{relativize ./url}}}">{{./displayVersion}}</a>
```

The div with the title could also be embedded in an if statement, and you won't have to duplicate the whole content of the page, which would obviously be way better ..